portal/src/presentation/markdownRenderer.ts

```diff
@@ -51,29 +51,75 @@ function jokulList({ ordered, children }: ListProps) {
 }
 
 function jokulHeading({ level, children }: HeadingProps) {
-    let Element: FunctionComponent;
+    let Element: ReactElement;
     switch (level) {
         case 1:
-            Element = H1;
+            Element = createElement(
```

Comment:
Får litt lyst til å lage en liiiiten fabrikk her.....

```
const createJklElement = (semanticElement, className, children) => {
  return createElement(
    `${semanticElement}`,
    {
         className
     }
     children,
};


...

case 1: 
  Element = createJklElement("h1", "jkl-title-large", children);
...
```

Reply:
Hehe, ja dette er skikkelig stygt. Men som nevnt over er dette egentlig en del av arbeidet Eduardo holder på med nå, så har bare kjørt rene erstatninger i Portalen for at det skal bygge. Kan samkjøre med ham, og evt. legge inn stilene han jobber med i denne PR-en.